docs/getting-started/install/dreamhost.md

```diff
@@ -0,0 +1,160 @@
+# Install Dokku on DreamHost Cloud Server with cloud-init
+
+Cloud-init script can be used to automate installation of Dokku on
+Dreamhost (or any other OpenStack-compatible cloud with minimal
+changes).
+
+To create a new server and install Dokku from the openstack command
+line interface:
+
+    $ openstack server create \
+       --image Ubuntu-16.04 \
+       --flavor gp1.semisonic \
+       --security-group default \
+       --key-name $YOUR_SSH_KEYNAME \
```

Comment:
Should this be called out somewhere? Is this a path to a key on a disk, or is it a registered key in an openstack service somewhere?